.circleci/config.yml

```diff
@@ -162,6 +193,8 @@ workflows:
   integration_tests:
     jobs:
       - e2e_tests
+      - prerender
```

Comment:
Looks like this was named `prerender-check` in `jobs` list above.

Reply:
Fixed, thanks!